projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts

```diff
@@ -78,21 +78,25 @@ export class ResourceViewComponent implements OnInit, OnChanges, OnDestroy {
         private _valueOperationEventService: ValueOperationEventService) { }
 
     ngOnInit() {
-        // subscribe to the event bus and listen for the ValueAdded event to be emitted
-        // when a ValueAdded event is emitted, get the resource again to display the newly created value
+        // subscribe to the ValueOperationEventService and listen for an event to be emitted
         this.valueOperationEventSubscription = this._valueOperationEventService.on(
```

Comment:
see my comment relating to subscriptions

Reply:
fixed in dda33f8